editor/src/components/canvas/canvas-strategies/absolute-reparent-strategy.tsx

```diff
@@ -62,15 +63,15 @@ export const absoluteReparentStrategy: CanvasStrategy = {
       show: 'visible-only-while-active',
     },
   ],
-  fitness: (canvasState, interactionState) => {
-    if (
-      canvasState.selectedElements.length > 0 &&
-      interactionState.activeControl.type === 'BOUNDING_AREA' &&
-      interactionState.interactionData.type === 'DRAG' &&
-      interactionState.interactionData.modifiers.cmd &&
-      interactionState.interactionData.drag != null
-    ) {
-      return 2
+  fitness: (canvasState, interactionState, strategyState) => {
+    // All 4 reparent strategies use the same fitness function findReparentStrategy
```

Comment:
This is a really neat idea to avoid code duplication in the different fitness functions